src/index.js

```diff
@@ -218,24 +218,20 @@ async function templateDir(from, to, useTS) {
  * @param {ConfigOptions} opts
  */
 async function installDeps(to, packageManager, opts) {
-	await installPackage(['preact'], { packageManager, cwd: to });
+	const dependencies = [];
+	const devDependencies = [];
 
-	if (opts.useTS) {
-		await installPackage(['typescript'], { packageManager, cwd: to, dev: true });
+	const installOpts = {
+		packageManager,
+		cwd: to,
+		silent: true,
```

Comment:
#16 changed installer, which defaults to non-silent installs. This simply restores the old behavior (and looks much cleaner).